open-sphere-base/control-panels/src/main/java/io/opensphere/controlpanels/toolbox/DefaultDetailPanelProvider.java

```diff
@@ -16,6 +16,7 @@
 /**
  * The default detail panel provider, used to provide detail panels for all data
  * types.
+ * @param <T>
```

Comment:
the @param <T> isn't needed here, as the generic type isn't declared in this class.